### PR TITLE
Update sql-server-connector-maintenance-troubleshooting.md

### PR DESCRIPTION
**Pull request by PhilB-MSFT**
Update to doc from customer feedback for case 2203210050001033.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@PhilB-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ShannonLeavitt** [comment]
@Rupp29 - Can you review the proposed changes? IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

Note: before this PR is merged, the new link needs to be updated to point to the [live article](https://docs.microsoft.com/azure/key-vault/general/access-behind-firewall) rather than the article in main.

#label:"aq-pr-triaged"
#assign:@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger18** [comment]
The following users don't have valid GitHub IDs, or are not collaborators on this repo: microsoftdocs/public-repo-pr-review-team

**Comment by Rupp29** [comment]
The proposed change looks good, but I can't update the page to the live URL

File: docs/relational-databases/security/encryption/sql-server-connector-maintenance-troubleshooting.md [hunk 1]
```diff
@@ -168,6 +168,11 @@ Key backups can be restored across Azure regions, as long as they remain in the
 - login.microsoftonline.com/*:443
 - *.vault.azure.net/*:443
 
+Additionally, checking the certificate revocation list may create http traffic on port 80. 
+
+ > [!NOTE]
+ > Using the SQL Server Connector for Azure Key Vault behind a firewall or proxy server can impact peformance if traffic is delayed or blocked. Familiarize yourself with [Access Azure Key Vault behind a firewall](https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/key-vault/general/access-behind-firewall.md) to ensure the correct rules are in place.
```
Comment (ShannonLeavitt) [inline docs/relational-databases/security/encryption/sql-server-connector-maintenance-troubleshooting.md:174]:
```suggestion
 > Using the SQL Server Connector for Azure Key Vault behind a firewall or proxy server can affect performance if traffic is delayed or blocked. Familiarize yourself with [Access Azure Key Vault behind a firewall](/azure/key-vault/general/access-behind-firewall) to ensure the correct rules are in place. 

```

**Comment by PRMerger15** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/relational-databases/security/encryption/sql-server-connector-maintenance-troubleshooting.md [hunk 2]
```diff
@@ -168,6 +168,12 @@ Key backups can be restored across Azure regions, as long as they remain in the
 - login.microsoftonline.com/*:443
 - *.vault.azure.net/*:443
 
+Additionally, checking the certificate revocation list may create http traffic on port 80.
```
Comment (ShannonLeavitt) [inline docs/relational-databases/security/encryption/sql-server-connector-maintenance-troubleshooting.md:171]:
```suggestion
Additionally, checking the certificate revocation list might create HTTP traffic on port 80. 

```
Style edits

**Comment by PRMerger6** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Rupp29** [comment]
@ShannonLeavitt : The style and content looks good to me. If you can update the URL to the live URL. I'm not able to update it myself.

**Pull request merged**